deploy/install-prereq.sh

```
@@ -60,8 +60,62 @@ function prompt_for_input() {
 
 function installing_prerequisites() {
 
-  echo "Creating esignet-global configmap in esignet namespace"
-  kubectl -n $NS apply -f esignet-global-cm.yaml
+  echo "Provide domain values (press Enter to accept sandbox defaults)"
+  MOSIP_API_HOST=${MOSIP_API_HOST:-}
+  if [ -z "$MOSIP_API_HOST" ]; then
+    read -p "mosip-api-host [api.sandbox.xyz.net]: " MOSIP_API_HOST
```

Shellcheck (0.11.0)

[info] read without -r will mangle backslashes.

(SC2162)